staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1241,28 +1253,33 @@ type cacheWatcher struct {
 	// if this field holds the value of 0 that means we don't have any special preferences toward delivering bookmark events
 	bookmarkAfterResourceVersion uint64
 
-	// wasBookmarkAfterResourceVersionSentMutex protects wasBookmarkAfterResourceVersionSent
-	wasBookmarkAfterResourceVersionSentMutex sync.Mutex
+	// sateMutex protects state
+	sateMutex sync.Mutex
```

Comment:
We could consider using atomic instead too - but I don't have strong preference. Mutex+int works for me too.